cmd/root.go

```diff
@@ -41,12 +42,31 @@ func Execute() {
 }
 
 func init() {
-	RootCmd.PersistentFlags().StringVarP(&rootConfig.kubeConfig, "kubeconfig", "c", "", "config file (default is $HOME/.kube/config")
+	cobra.OnInitialize(processFlags)
+	RootCmd.PersistentFlags().BoolVarP(&rootConfig.localMode, "local", "l", false, "[DEPRECATED] Local mode, uses $HOME/.kube/config as configuration")
+	RootCmd.Flags().MarkHidden("local")
+
```

Comment:
s/\\n//

Reply:
Resolved in https://github.com/Shopify/kubeaudit/pull/113/commits/9c8b9b6a7958a9338e757e4e25fe8f295b217957.